### PR TITLE
Add support for multiple prompts within a single question.

### PR DESCRIPTION
**Pull request by seanlip**
/cc @apicardgoog

### CODE REVIEW

File: client/components/LearnerViewDirective.js [hunk 1]
```diff
@@ -24,24 +24,41 @@ tie.directive('learnerView', [function() {
       <h3>Exercise: {{title}}</h3>
 
       <div class="tie-learner-view-left-column">
+        <div class="tie-old-instructions">
```
Comment (eyurko) [inline client/components/LearnerViewDirective.js]: I'm not sure what the difference is between old-instructions and instructions. Could you clarify?

File: client/components/LearnerViewDirective.js [hunk 2]
```diff
@@ -105,25 +133,49 @@ tie.directive('learnerView', [function() {
...
         var question = QuestionDataService.getData();
         var prompts = question.getPrompts();
+        var currentPromptIndex = 0;
 
-        var currentPrompt = prompts[0];
-        $scope.title = question.getTitle();
-        $scope.code = question.getStarterCode(language);
-        $scope.instructions = currentPrompt.getInstructions();
-        $scope.feedback = '';
+        $scope.questionIsCompleted = false;
```
Comment (eyurko) [inline client/components/LearnerViewDirective.js]: So here you're using questionIsCompleted, and elsewhere you use isAnswerCorrect() and isLastPrompt(). Is this just a stylistic difference to differentiate between boolean variables and methods that return booleans?

File: client/components/LearnerViewDirective.js (hunk 1 above)
Comment (eyurko) [inline client/components/LearnerViewDirective.js]: Oh no wait I got it. Might be worth a comment, or calling them "previous-instructions"?

File: client/components/LearnerViewDirective.js (hunk 2 above)
Comment (seanlip) [inline client/components/LearnerViewDirective.js]: Yep, that's the intention.

File: client/components/LearnerViewDirective.js (hunk 1 above)
Comment (seanlip) [inline client/components/LearnerViewDirective.js]: Yep, I can change it to "previous-instructions". I'll do that shortly.

Comment (seanlip) [inline client/components/LearnerViewDirective.js]: Done.

File: client/components/LearnerViewDirective.js (hunk 2 above)
Comment (eyurko) [inline client/components/LearnerViewDirective.js]: Cool, just wanted to make sure for my own sake.

**Pull request merged**